tests/integration/test_mender_mcu.py

```
@@ -16,8 +16,9 @@
 import time
 import pytest
 import helpers
+import logging
 
-from mender_integration.tests.MenderAPI import logger
+logger = logging.getLogger(__name__)
 
```

Below here, at:

```
@pytest.fixture
def teardown():
    yield
    os.remove(helpers.get_header_file())
```

The test will fail if the file was not created:
```
====================================== ERRORS ======================================
____________________ ERROR at teardown of test_deployment_abort ____________________

    @pytest.fixture
    def teardown():
        yield
>       os.remove(helpers.get_header_file())
E       FileNotFoundError: [Errno 2] No such file or directory: '/home/lluis/west/mender-mcu-integration/tests/integration/src/test_definitions.h'

test_mender_mcu.py:32: FileNotFoundError
```

We should check that the file exists before remove (or just try remove and then except pass...)